source/includes/steps-atlas-cli-deploy-local-prereqs.rst

```diff
@@ -0,0 +1,56 @@
+
+.. procedure:: 
+   :style: normal 
+
+   .. step:: Install the dependencies.
+
+      a. Install the :ref:`{+atlas-cli+} <install-atlas-cli>`.
+
+         If you use `Homebrew <https://brew.sh/#install>`__, you can
+         run the following command in your terminal:
+
+         .. code-block::
+
+            brew install mongodb-atlas-cli
+               
+         For install instructions on other operating systems,
+         see :ref:`install-atlas-cli`.
+
+      #. Install `Docker <https://www.docker.com/>`__.
+
+         - For MacOS or Windows, install `Docker Desktop v4.31+ <https://docs.docker.com/desktop/release-notes/#4310>`__. 
+         - For Linux, install `Docker Engine v27.0+ <https://docs.docker.com/engine/release-notes/27.0/>`__.
+
+         .. note::
```

Comment:
I would get rid of the note and move the first sentence about Docker to be right after the sentence on line 19 (where the step is). 
I would then move the second sentence to be the third bullet.

Reply:
Great suggestion!